app/components/telegram_widget/telegram_widget.css

```diff
@@ -0,0 +1,3 @@
+.TelegramWidget {
+  margin-top: 2rem;
+}
```

Comment:
This is an example for what I tried to explain when we spoke about presentational components on Tuesday and trying to make components context-independent.

The goal in this case is that there is some whitespace between the Telegram widget and the list that is rendered before.

So the context of this is actually a parent element of the widget and a list -- the white space should be between the two of them.

By adding the whitespace to the `TelegramWidget` component, the component becomes dependent on its context. Also, anytime you change the spacing between elements in the `OnboardingInstructions` component, you also need to adjust them in the `TelegramWidget` component, which is an indicator for tight coupling.

Now this might not be a huge problem in this case, and it probably won’t become a problem in the future, as the `TelegramWidget` component is likely going to be a one-off component, so the possible implications I mentioned above might be a bit constructed in this case.

But in general, I think it increases maintainability to reduce those kinds of side effects.

Reply:
In this case, you could actually leverage the `Stack` component that is already being used to add the spacing between the bold text and the list. A stack adds whitespace between its child elements.

<img width="360" alt="Screen Shot 2020-10-31 at 12 52 11" src="https://user-images.githubusercontent.com/1512805/97778786-d268ac00-1b79-11eb-96f4-3bfe58a77746.png">

Reply:
I understand your logic, and I agree that if we decide to reuse, we might need to update the styling so it can be used in different contexts. I disagree that adding html elements to add this styling is a good direction.

The way I would prefer to solve this is to target the `TelegramWidget` in CSS in a way that is namespaced, so that it would only affect the styling in that one instance. I have added a commit https://github.com/tactilenews/100eyes/pull/440/commits/db962391c922e89b0de681e64bb40701399ace07 for that.